client/me/account/hosting-dashboard-opt-in-form.tsx

```diff
@@ -93,6 +105,34 @@ export default function HostingDashboardOptInForm() {
 							<span>{ translate( 'I want to try the beta version.' ) }</span>
 						</FormLabel>
 					</FormFieldset>
+					{ showOptOutSurvey && (
+						<FormFieldset className="account__hosting-dashboard-notice-banner">
+							<NoticeBanner
+								title={ translate( 'Prefer the previous version?' ) }
+								level="info"
+								hideCloseButton
+							>
+								{ translate(
+									'{{surveyLink}}Please complete this short survey{{/surveyLink}} to help us understand what didn’t work and how we can improve.',
+									{
+										components: {
+											surveyLink: (
+												<ExternalLink
+													href="https://automattic.survey.fm/new-hosting-dashboard-opt-out-survey"
+													icon
+													onClick={ () =>
+														recordTracksEvent(
```

Comment:
I didn't see this event being triggered, might need to wrap it in a `dispatch()`.